### ISSUE TITLE
What is the recommended number of iterations for replacing the top layer?

### ISSUE THREAD

**Issue by DesBw** [Shreeshrii/tess5train-fonts#20]
> [engLayer](https://github.com/Shreeshrii/tess5train-fonts#englayer)
> Replace the top layer of network of tessdata_best/eng.traineddata for adding multiple characters such as superscripts, fraction symbols, etc using multiple fonts which support the characters. Evaluation done on data using the same fonts.
> 
> For replacing the top layer, we will cut off the last LSTM layer and the softmax, replacing with a smaller LSTM layer and a new softmax.
> 
> **For a new language, it is possible to cut off the top layers of an existing network and train, as if from scratch, but a fairly large amount of training data is still required to avoid over-fitting.**
> 

That is very interesting note. I was not able to find any documentation on the impact of top-layer training on the base model. 

Still, the question is how much is the base model affected during the top layer training?

I know the rule of thumb for fine-tuning is to keep the iterations less than 400. That is to avoid tampering the base model. Does the same kind of recommendation apply for training by removing the top layer?
